### PR TITLE
Problem lists

### PR DESCRIPTION
**Pull request by AnikaC-git**
## Checklist for submitter & reviewer

- [x] copy constructor copies the primary key Id of the entity
- [x] copy constructor copies all other fields of the entity, starting with `super(other)`
- [x] unique constraints added if appropriate
- [x] text definition added to columns which could have more than 255 characters
- [x] foreign key names are in the form `<entityClass>Id`, and no typos
- [x] during run, glowroot is checked for query speed and indexes added if needed
- [x] validation run completed and validated

### CODE REVIEW

File: inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/conditions/ConditionType.java [hunk 1]
```diff
@@ -67,7 +67,7 @@
     /**
      * Minimal information constructor.
      * @param dataType   Type of patient state type; either patient infection or problem list
-     * @param code       EPIC code of the patient state type
+     * @param code       code of the patient condition type
```
Comment (stefpiatek) [inline inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/conditions/ConditionType.java:70]: this isn't you, but we're not setting the conditionTypeId in the copy constructor

**Review by stefpiatek — APPROVED** [review]
Thanks for updating the documentation

File: inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/conditions/PatientCondition.java [hunk 2]
```diff
@@ -105,7 +105,7 @@ public class PatientCondition extends TemporalCore<PatientCondition, PatientCond
     private String status;
 
     /**
-     * \brief Problem List priority.
+     * \brief Problem priority.
      */
     private String priority;
```
Comment (t-young31) [inline inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/conditions/PatientCondition.java:110]: Document for this and other fields which are expected to be not null

Comment (t-young31) [inline inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/conditions/PatientCondition.java:110]: Document for this and other fields which are expected to be not null

**Review by skeating — APPROVED** [review]

**Pull request merged**